packages/cfd/src/features/Containers/trade-modals/other-cfds-trade-modal.tsx

```diff
@@ -95,7 +96,7 @@ const OtherCFDsTradeModal = ({
                     <Text
                         align='center'
                         as='p'
-                        className='cfd-trade-modal__download-center-text'
+                        className='trade-modal__download-center-text'
```

Comment:
```suggestion
                        className='trade-modal__download__tect--center'
```

Reply:
this is referring to the `text` in `download-center` , rather than positioning `download__text` at `center`